go.mod

```diff
@@ -346,3 +346,5 @@ require (
 )
 
 replace github.com/go-micro/plugins/v4/store/nats-js => github.com/kobergj/plugins/v4/store/nats-js v1.2.1-0.20231020092801-9463c820c19a
+
+replace github.com/cs3org/reva/v2 => github.com/2403905/reva/v2 v2.0.0-20231115102502-6f73f237c62f
```

Comment:
DO NOT MERGE. The reva replacement have to be removed. 
The CI is green only with the reva fork https://drone.owncloud.com/owncloud/ocis/28779